policyengine_canada/variables/gov/provinces/ns/tax/income/credits/nsprc/ns_poverty_reduction_credit.py

```diff
@@ -0,0 +1,20 @@
+from policyengine_canada.model_api import *
+
+
+class ns_poverty_reduction_credit(Variable):
+    value_type = float
+    entity = Household
+    label = "Nova Scotia Poverty Reduction Credit"
+    unit = CAD
+    definition_period = YEAR
+    reference = (
+        "https://novascotia.ca/coms/noteworthy/PovertyReductionCredit.html"
+    )
+    defined_for = ProvinceCode.NS
+
+    def formula(household, period, parameters):
+        income = household("adjusted_family_net_income", period)
+        children = household("count_children", period)
+        p = parameters(period).gov.provinces.ns.tax.income.credits.nsprc
+        eligible = (children == 0) & (income < p.income_threshold)
```

Comment:
What about the Income Assistance requirement?

Reply:
Pavel and I have created an empty ns_assistance_requirement variable, and we will create this program in the next pull request